src/sagemaker/tensorflow/model.py

```diff
@@ -268,8 +268,9 @@ def deploy(
         kms_key=None,
         wait=True,
         data_capture_config=None,
-        update_endpoint=None,
+        async_inference_config=None,
         serverless_inference_config=None,
+        update_endpoint=None,
```

Comment:
any specific reason to change the sequence of args? This won't be backwards compatible for existing users of tensorflow model (users who have update_endpoint set to value, not using named arguments) and would require them to update their code.

Reply:
Bad changes during resolving conflicts. Will change it back